packages/babel-plugin/src/keyframes/__tests__/call-expression.test.tsx

```diff
@@ -0,0 +1,1157 @@
+import { transform } from '../../__tests__/test-utils';
+import {
+  longhandCssPropObjectCallExpression,
+  longhandCssPropTaggedTemplateExpression,
+  longhandStyledObjectCallExpression,
+  longhandStyledTaggedTemplateExpression,
+  shorthandCssPropObjectCallExpression,
+  shorthandCssPropTaggedTemplateExpression,
+  shorthandStyledObjectCallExpression,
+  shorthandStyledTaggedTemplateExpression,
+} from '../__fixtures__';
+
+describe('keyframes', () => {
+  beforeAll(() => {
+    process.env.AUTOPREFIXER = 'off';
+  });
+
+  afterAll(() => {
+    delete process.env.AUTOPREFIXRER;
+  });
+
+  describe('transforms an object call expression', () => {
+    const createSmokeTest = (usage: string) => `
+      import { css, keyframes, styled } from '@compiled/react';
+
+      const fadeOut = keyframes({
+        'from, 25%': {
+          opacity: 1,
+        },
+        '50%': {
+          opacity: 0.5,
+        },
+        to: {
+          opacity: 0,
+        },
+      });
+
+      ${usage}
+    `;
+
+    describe('referenced through a css prop object call expression', () => {
+      describe('by inlining static values', () => {
+        it('used with the longhand syntax', () => {
+          const actual = transform(createSmokeTest(longhandCssPropObjectCallExpression));
+
+          expect(actual).toMatchInlineSnapshot(`
+            "const _4 = \\"._1pgl1ytf{animation-timing-function:ease-in-out}\\";
+            const _3 = \\"._j7hq18s7{animation-name:k6qtb99}\\";
+            const _2 = \\"@keyframes k6qtb99{25%,0%{opacity:1}50%{opacity:0.5}to{opacity:0}}\\";
```

Comment:
nit not sure if we have control on it, but should we use `100%` instead of `to`?

Reply:
postcss will convert any input I provide to the shortest form. So in this case, `from` becomes `0%` and `to` stays as `to`